bks/map/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import *
+
+class FacilitySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Facility
+        fields = '__all__'
```

Comment:
In the POSIX standard, a line is defined as: A sequence of zero or more non- characters plus a terminating character. We should follow this standard and add a newline at the end of a file.